src/main/java/com/icthh/xm/ms/entity/config/RetryInterceptorConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package com.icthh.xm.ms.entity.config;
+
+import org.springframework.cloud.client.loadbalancer.LoadBalancerAutoConfiguration;
+import org.springframework.cloud.client.loadbalancer.RestTemplateCustomizer;
+import org.springframework.cloud.client.loadbalancer.RetryLoadBalancerInterceptor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import java.util.ArrayList;
+
+@Configuration
+public class RetryInterceptorConfiguration extends LoadBalancerAutoConfiguration.RetryInterceptorAutoConfiguration {
+
+    @Bean
+    @Profile("nolb")
```

Comment:
do we use this profile? maybe let's remove it @osiievska23

Reply:
done